fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -255,15 +255,16 @@ fun rememberDrawerState(confirmValueChange: (DrawerValue) -> Boolean = { true })
 
 @Composable
 fun rememberBottomDrawerState(
+    initialState: DrawerValue = DrawerValue.Closed,
     expandable: Boolean = true,
     skipOpenState: Boolean = false,
     confirmValueChange: (DrawerValue) -> Boolean = { true }
 ): DrawerState {
     return rememberSaveable(
-        confirmValueChange, expandable, skipOpenState,
+        initialState, confirmValueChange, expandable, skipOpenState,
         saver = DrawerState.Saver(expandable, skipOpenState, confirmValueChange)
     ) {
-        DrawerState(DrawerValue.Closed, confirmValueChange, expandable, skipOpenState)
+        DrawerState(initialState, confirmValueChange, expandable, skipOpenState)
```

Comment:
since initialState and initalValue are the same thing, can we follow same naming everywhere to avoid confusion

Reply:
Updated the same